test/unit/legacy_wrappers/db.test.js

```diff
@@ -0,0 +1,284 @@
+'use strict';
+
+const {
+  MongoClient,
+  ListCollectionsCursor,
+  ChangeStream,
+  AggregationCursor,
+  Collection
+} = require('../../../src/index');
+const mongodbDriver = require('mongodb');
+const { Db } = require('mongodb');
+const sinon = require('sinon');
+const { expect } = require('chai');
+
+describe('legacy_wrappers/db.js', () => {
+  let db;
+  let client;
+  beforeEach(async function () {
+    client = new MongoClient('mongodb://iLoveJs');
+    db = client.db();
+  });
+
+  afterEach(async function () {
+    sinon.restore();
+    await client.close();
+  });
+
+  it('should return legacy listCollections cursor', () => {
+    expect(db.listCollections()).to.be.instanceOf(ListCollectionsCursor);
+  });
+
+  it('should return legacy ChangeStream', () => {
+    expect(db.watch()).to.be.instanceOf(ChangeStream);
+  });
+
+  it('should return legacy ChangeStream', () => {
+    expect(db.aggregate()).to.be.instanceOf(AggregationCursor);
+  });
+
+  it('should support db.addUser(username, callback)', done => {
```

Comment:
can we similarly improve these `it` statements to be more descriptive of what's actually being tested? clearly indicate success/failure expectations and all that jazz? (see my long comment on api.test.js)